gallia-prehistoire.csl

```diff
@@ -0,0 +1,347 @@
+            <group>
+              <label variable="page" form="short"/>
+              <text variable="page" prefix="&amp;#8239;"/>
+            </group>
+          </if>
+          <else>
+            <text variable="URL" text-decoration="underline"/>
+          </else>
+        </choose>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <group delimiter=", " prefix="(coll. " suffix=")">
+      <text variable="collection-title" font-weight="normal"/>
+      <text variable="collection-number"/>
+    </group>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if type="bill book chapter graphic legal_case legislation motion_picture song report" match="any">
+        <choose>
+          <if match="any" is-numeric="edition">
+            <group delimiter=" ">
+              <number variable="edition" form="ordinal"/>
+              <text term="edition" form="short"/>
+            </group>
+          </if>
+          <else>
+            <text variable="edition" text-case="capitalize-first" font-variant="normal" suffix=", "/>
+          </else>
+        </choose>
+        <text macro="issue" prefix=", "/>
+      </if>
+      <else-if type="article-magazine article-newspaper" match="any">
+        <group delimiter=" ">
+          <text macro="voljournal" font-weight="normal"/>
+        </group>
+      </else-if>
+      <else-if type="article article-journal" match="any">
+        <group>
+          <text macro="voljournal" text-case="capitalize-first" font-style="normal"/>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="issue">
+    <choose>
+      <if match="any" is-numeric="issue">
+        <text term="issue" form="short" prefix="(" suffix=")"/>
+        <text variable="issue"/>
+      </if>
+      <else>
+        <text variable="issue"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="voljournal">
+    <group delimiter=" ">
+      <text term="volume" form="short"/>
+      <text variable="volume"/>
+      <text term="issue" form="short"/>
+      <text variable="issue"/>
+    </group>
+  </macro>
+  <macro name="author-opcit">
+    <choose>
+      <if match="any" variable="author">
+        <names variable="author" delimiter=", ">
+          <name form="short" and="text" delimiter-precedes-last="never" et-al-min="3" et-al-use-first="1" initialize-with="." sort-separator=" "/>
+          <et-al font-style="italic"/>
+        </names>
+      </if>
+      <else-if match="any" variable="editor">
+        <names variable="editor" delimiter=", ">
+          <name and="text" delimiter-precedes-last="never" et-al-min="3" initialize-with="." sort-separator=" "/>
+          <et-al/>
+        </names>
+        <text term="editor" prefix=" "/>
+      </else-if>
+    </choose>
+    <text term="cited" font-style="italic" prefix=", "/>
+    <text variable="locator" prefix="p.&amp;#160;"/>
+  </macro>
+  <citation and="text" et-al-min="3" et-al-use-first="1" sort-separator=" ; ">
```

Comment:
Note also that the "et-al" attributes on `<citation>` (and `<bibliography`>) trickle down to the `<names>` elements. The latter override the former.